backend/docker-compose.yml

```
@@ -1,75 +1,61 @@
-      - DB_USER=${POSTGRES_USER}
-      - DB_PASSWORD=${POSTGRES_PASSWORD}
-      - DB_HOST=${POSTGRES_HOST}
-      - DB_PORT=${POSTGRES_PORT}
-    networks:
-      - app-network
-    tty: true
-    stdin_open: true
+    depends_on:
+      - postgres
+      - redis-stack
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)